lib/ffi/aspell/aspell.rb

```diff
@@ -13,8 +16,21 @@ module FFI
   # For more information see {FFI::Aspell::Speller}.
   #
   module Aspell
-    extend   FFI::Library
-    ffi_lib ['aspell', 'libaspell.so.15']
+    extend FFI::Library
+
+    begin
+      stdout, stderr, status = ::Open3.capture3("brew", "--prefix")
+      homebrew_path  = if status.success?
+                        "#{stdout.chomp}/lib"
+                      else
+                        '/usr/local/homebrew/lib'
+                      end
+    rescue
+      # Homebrew doesn't exist
+    end
+
+    ffi_lib ['aspell', 'libaspell.so.15'] if ::RbConfig::CONFIG['host_os'] =~ /linux/
+    ffi_lib ["#{homebrew_path}/libaspell.dylib"] if ::RbConfig::CONFIG['host_os'] =~ /darwin/
```

Comment:
Yeah, I'm not a big fan of it either (running a script to determine homebrew location is particularly icky) but I've been looking through how other FFI gems have done it and this seems like the most common way.

Have you any other suggestions?